src/Components/Config/ConfigScreen.js

```diff
@@ -145,28 +145,30 @@ class ConfigScreen extends PureComponent {
     if (validation) {
       let isValid = true
     
-      if (!validation.canBeBlank && value == '') {
-        isValid = false
-      }
-    
-      if (validation.allowedChars && !value.match(validation.allowedChars)) {
-        isValid = false
-      }
-    
-      if (validation.format && !value.match(validation.format)) {
-        isValid = false
-      }
+      if (value !== null) {
+        if (!validation.canBeBlank && value == '') {
```

Comment:
Similarly, might be worth checking the table for string coercion in an `if`

Reply:
That's preexisting code, but it seems that it'd only succeed if value was `false` and `0`. `null` and `undefined` both do not equal (unstrict) `''`

I'm thinking this check should be strict equality; I'd expect `canBeBlank` is referring to strings